spec/service_worker/index.bs

```diff
@@ -152,6 +157,7 @@ spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
         text: potential-navigation-or-subresource request
         text: process response
         text: process response end-of-file
+        text: read a chunk from a reader; url: concept-read-chunk-from-reader
```

Comment:
Yeah,  I will create a separate PR for the fetch spec. Same for concept-cancel-stream-reader (see [TODO](https://github.com/slightlyoff/ServiceWorker/pull/934/#issuecomment-234461814))